<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1484</width>
    <height>817</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1484</width>
    <height>817</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1484</width>
    <height>817</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>智能任务提醒</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>hnnd.ico</normaloff>hnnd.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>20</y>
      <width>1471</width>
      <height>661</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_0">
     <attribute name="title">
      <string>      服务器设置      </string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>511</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="serveraddress">
         <property name="text">
          <string>服务器地址</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="uip_ip"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="serverport">
         <property name="text">
          <string>端口号码</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="uip_port"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="databasename">
         <property name="text">
          <string>数据库名称</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="uip_dbname"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="account">
         <property name="text">
          <string>MySQL USER</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="uip_dbacc"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="password">
         <property name="text">
          <string>MySQL PW</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="uip_pw"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>              项目状态查询              </string>
     </attribute>
     <widget class="QTableView" name="tableView_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>1241</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
     </widget>
     <widget class="QPushButton" name="sl_querybtn">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>69</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sl_querybtn_2">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>159</y>
        <width>171</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>新增记录</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sl_querybtn_3">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>249</y>
        <width>171</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sl_querybtn_4">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>340</y>
        <width>171</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="filter_cname">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>客户姓名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="cname">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="ncuinfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>1241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QLabel" name="filter_cname_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>19</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>项目进度</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="add_process">
      <property name="geometry">
       <rect>
        <x>618</x>
        <y>19</y>
        <width>230</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>请选择</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>预受理</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>调查走访</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>评审流程</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>合同签订</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>批复变更</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>银行信息变更</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>核保放款</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>确认放款</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>循环项目银行信息变更</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>循环项目核保放款</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>循环项目确认放款</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>项目展期</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>委托代偿</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>被动代偿</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>项目还款</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>              已完结项目状态              </string>
     </attribute>
     <widget class="QPushButton" name="sl_querybtn_5">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>69</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>1241</width>
        <height>381</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
     </widget>
     <widget class="QLabel" name="filter_cname_2">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>客户姓名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="cname_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sl_querybtn_7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>249</y>
        <width>171</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sl_querybtn_8">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>340</y>
        <width>171</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sl_querybtn_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>159</y>
        <width>171</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>新增记录</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="ccuinfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>1241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QLabel" name="filter_cname_4">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>19</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>项目进度</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="add_process_2">
      <property name="geometry">
       <rect>
        <x>618</x>
        <y>19</y>
        <width>230</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>请选择</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B类资料归档</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>项目在保</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>项目解保</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>项目终结</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>          数据统计          </string>
     </attribute>
     <widget class="QTableView" name="tableView_X">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>1251</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ff2761;&quot;&gt;在保额合计：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="zbcr">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>— —</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="zbcc">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>— —</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>30</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#63dbff;&quot;&gt;在保客户数：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>100</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>在保额统计表(分产品)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>572</x>
      <y>730</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Smart TaskReminder &lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#ff0000;&quot;&gt;v0.7.3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>473</x>
      <y>690</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>当前状态：就绪</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1484</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actiond"/>
    <addaction name="actione"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actiond">
   <property name="text">
    <string>重新连接服务器</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actione">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>版权信息</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
